### PR TITLE
Improve linting

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

**Comment by franmomu** [comment]
There are some `yaml` warnings in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2283/files

**Comment by phansys** [comment]
>There are some `yaml` warnings in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2283/files

Fixed.

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -16,13 +16,68 @@ jobs:
...
+        uses: actions/checkout@v2
+
+      - name: Install required dependencies
+        run: sudo apt-get update && sudo apt-get install libxml2-utils
+
+      - name: Lint xml files
+        run: make lint-xml
+
+  yaml-files:
+    name: YAML files
```
Comment (franmomu) [inline .github/workflows/coding-standards.yml:66]:
```suggestion
  link-yaml-files:
    name: Lint YAML files
```

File: .github/workflows/coding-standards.yml [hunk 2]
```diff
@@ -16,13 +16,68 @@ jobs:
...
+
+      - name: Install Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
+
+      - name: Install required gem
+        run: gem install yaml-lint
+
+      - name: Lint files
```
Comment (franmomu) [inline .github/workflows/coding-standards.yml:82]:
```suggestion
      - name: Lint yml files
```

**Review by franmomu — APPROVED** [review]
Just some comments for consistency with the other jobs

**Comment by phansys** [comment]
>Just some comments for consistency with the other jobs

I agree. Comments addressed.

**Review by franmomu — APPROVED** [review]

**Comment by franmomu** [comment]
thanks @phansys!

**Pull request merged**